### ISSUE TITLE
Is possible use Playwright instead of Puppeteer?

### ISSUE THREAD

**Issue by galegobr01** [thomasdondorf/puppeteer-cluster#326]
Is possible to use [Playwrigth](https://github.com/microsoft/playwright) instead of Puppeteer?

**Comment by Discartyptics** [comment]
I too may be interested in this possibility! 
I was considering  moving to *Playwright* but **Cluster** is so good I don't think I can make the move without it.

**Comment by thomasdondorf** [comment]
Yes, this looks interesting. It probably won't be too much work to implement it.
